test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -244,3 +277,29 @@ func setupPathForTest(t *testing.T) func() {
 		}
 	}
 }
+
+// Retry to ensure CR has been created for up to a timeout and then
+// set the Ready Condition of CR to true
+func setCRReadyCondition(t *testing.T, cmCl versioned.Interface, crName, crNamespace string) {
+	timeout := time.After(5 * time.Minute)
+	tick := time.Tick(1 * time.Second)
+	for {
+		select {
+		case <-timeout:
+			t.Fatal("timeout waiting for CertificateRequest to be signed, retry later with fetch command")
+		case <-tick:
+			req, err := cmCl.CertmanagerV1alpha2().CertificateRequests(crNamespace).Get(context.TODO(), crName, metav1.GetOptions{})
+			if err != nil {
+				continue
+			}
+			// CR has been created, try update status
+			readyCond := cmapiv1alpha2.CertificateRequestCondition{Type: cmapiv1alpha2.CertificateRequestConditionReady, Status: cmmeta.ConditionTrue}
+			req.Status.Conditions = []cmapiv1alpha2.CertificateRequestCondition{readyCond}
+			_, err = cmCl.CertmanagerV1alpha2().CertificateRequests(crNamespace).UpdateStatus(context.TODO(), req, metav1.UpdateOptions{})
```

Comment:
Seems like we don't actually set `status.certificate` at all 🤔  I'd expect the CLI command to actually fail if it encounters a CR like this - it's never a valid state to be in.

Reply:
So if you could..

1) update the CLI to actually fail if it encounters this state
2) update the tests to exercise the case where no `status.certificate` is set as well as the case where it _is_ set (so we have full coverage there 😄)

Reply:
"it's never a valid state to be in", do you mean that the CR has the Ready condition set but `status.certificate` is empty? Should I be checking that case? I think we are assuming that the Status is controlled by the controllers so that shouldn't happen at all?

Also what should I put in `status.certificate`? Good idea to copy from some local CR I created?

Reply:
Oh man.. I always reply on an outdated page without refreshing and miss out on replies, ok I'll do that then